src/main/webapp/WEB-INF/view/profile.jsp

```diff
@@ -10,15 +10,31 @@ List<Message> messages = (List<Message>) request.getAttribute("messages");
 <html>
 	<head>
 		<title>Profile Page</title>
-	  <link rel="stylesheet" href="/css/main.css" type="text/css">
+	  <link rel="stylesheet" href="/css/main.css">
 	  <link rel="shortcut icon" href="/images/JavaChipsLogo.png" />
-
-
+		<link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+		<link rel="stylesheet" href="https://code.getmdl.io/1.3.0/material.cyan-yellow.min.css">
+		<script defer src="https://code.getmdl.io/1.3.0/material.min.js"></script>
 		<style>
-			#chat {
```

Comment:
Can you explain why you deleted this text area?

Reply:
Sent messages used to display in a white text box background. I removed this so that messages would just display in the regular colored background of the site. Thought it looked much nicer that way.